app/models/course/assessment/category.rb

```diff
@@ -8,10 +10,16 @@ class Course::Assessment::Category < ActiveRecord::Base
 
   accepts_nested_attributes_for :tabs
 
+  before_create :build_initial_tab
   before_destroy :validate_before_destroy
 
   default_scope { order(:weight) }
 
+  def self.after_course_create(course)
+    course.assessment_categories.
+      create(title: Course::Assessment::Category.human_attribute_name('title.default'), weight: 0)
```

Comment:
I'm a little concerned because I'm not sure if this would be in the required localisation.